### DISCUSSION TITLE
What prompt was used for instruct models?

### DISCUSSION THREAD

**Discussion by vprelovac** [embeddings-benchmark/mteb#258] [Q&A] [answered]
Was Instruct model tuned per task or was the default setting used?

**Comment by Muennighoff** [comment]
I think the instructor model on the leaderboard uses a different prompt for each task

**Comment by vprelovac** [comment]
Is there any reference to what is used?

**Comment by Muennighoff** [comment, marked as answer]
I think it's in their paper: https://arxiv.org/abs/2212.09741

**Comment by KennethEnevoldsen** [comment]
As this is a question I will convert it to a discussion thread instead.
